HangmanGame/Game.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HangmanGame
+{
+    public class Game : IDisposable
```

Comment:
What's the purpose of inheriting from `IDisposable`?

Reply:
My best conclusion is that its used so he could destroy the Game class cleanly? I don't believe it would actually be necessary, however, given the code should not have any locks on IO. What do you think?

Reply:
The `Game` class does not make use of any unmanaged resources, so I don't see the need to inherit from `IDisposable`.

`IDisposable` requires the implementation of the `Dispose()` method. In lines `137-138` below, you can see that the method doesn't do anything.

I suggest this class should not derive from `IDisposable` and that the `Dispose()` method be removed.